rotkehlchen/db/eth2.py

```diff
@@ -126,18 +126,21 @@ def get_validator_daily_stats_and_limit_info(
             self,
             cursor: 'DBCursor',
             filter_query: 'Eth2DailyStatsFilterQuery',
-    ) -> tuple[list[ValidatorDailyStats], int]:
+    ) -> tuple[list[ValidatorDailyStats], int, FVal]:
         """Gets all eth2 daily stats for the query from the DB
 
         Returns a tuple with the following in order:
          - A list of the daily stats
          - How many are the total entries found for the filter
+         - Sum of ETH gained/lost for the filter
         """
         stats = self.get_validator_daily_stats(cursor, filter_query=filter_query)
         query, bindings = filter_query.prepare(with_pagination=False)
-        query = 'SELECT COUNT(*) from eth2_daily_staking_details ' + query
-        count = cursor.execute(query, bindings).fetchone()[0]
-        return stats, count
+        # TODO: A weakness of this query is that it does not take into account the
+        # ownership proportion of the validator in the PnL here
```

Comment:
should this go into the docs or an issue? I believe it will affect the users that track profits for rocket eth validators

Reply:
Issue. I want to make one